src/Domain/Teams/Team.cs

```diff
@@ -6,7 +6,7 @@
 namespace SSW.CleanArchitecture.Domain.Teams;
 
 // For strongly typed IDs, check out the rule: https://www.ssw.com.au/rules/do-you-use-strongly-typed-ids/
```

Comment:
Did you update the rule for the 'sealed' use case?

Reply:
No I haven't updated the rule.  This change was a strange one.  The problem I've found is that sometimes you need to use optional/nullable foreign keys.  When you do this EF Core doesn't like trying to use nullable structs for FKs.  The only way I could find to fix this is to change from a `record struct` to `record`.

Should probably create a ticket to further investigte this and update the rule if necesary.

Reply:
Added a PBI to track - https://github.com/SSWConsulting/SSW.CleanArchitecture/issues/329#issue-2326636333